src/common/services/plays.js

```diff
@@ -133,7 +133,7 @@ export const deleteATag = (play_id, actualTags, newTags) => {
   return toBeDeletedTags; // array of promises to be resolved or rejected
 };
 
-export const getPlaysByFilter = async (filter) => {
+export const getPlaysByFilter = async (filter, sortBy) => {
```

Comment:
this parameter is being used anywhere. either use it or remove it

Reply:
This function uses `sortBy` param in line 193 of the same file.
Please check!